src/python-lxc/lxc.c

```diff
@@ -25,14 +25,91 @@
 #include <Python.h>
 #include "structmember.h"
 #include <lxc/lxccontainer.h>
-#include "lxc/utils.h"
-#include "lxc/namespace.h"
-#include "lxc/confile.h"
 #include <stdio.h>
 #include <sys/wait.h>
+#include <sched.h>
+
+/*
+ * CLONE_* definitions copied from lxc/namespace.h
+ */
+#ifndef CLONE_FS
+#  define CLONE_FS                0x00000200
+#endif
+#ifndef CLONE_NEWNS
+#  define CLONE_NEWNS             0x00020000
+#endif
+#ifndef CLONE_NEWCGROUP
+#  define CLONE_NEWCGROUP         0x02000000
+#endif
+#ifndef CLONE_NEWUTS
+#  define CLONE_NEWUTS            0x04000000
+#endif
+#ifndef CLONE_NEWIPC
+#  define CLONE_NEWIPC            0x08000000
+#endif
+#ifndef CLONE_NEWUSER
+#  define CLONE_NEWUSER           0x10000000
+#endif
+#ifndef CLONE_NEWPID
+#  define CLONE_NEWPID            0x20000000
+#endif
+#ifndef CLONE_NEWNET
+#  define CLONE_NEWNET            0x40000000
+#endif
+
+/* From sys/personality.h */
+#define PER_LINUX 0x0000
+#define PER_LINUX32 0x0008
```

Comment:
Have you confirmed we actually need those? ^

I believe we have them as ifndef in the LXC code tree because of the Android C library not defining those, but that shouldn't be a problem on any system shipping python.